desktop/src/containers/Clock/clockOutContainer.jsx

```diff
@@ -99,6 +99,7 @@ export class ClockOut extends Component {
               projectTaskObjects &&
               projectTaskObjects[projectTaskId] &&
               /Other/.test(projectTaskObjects[projectTaskId].task.name)
+              && formikProps.values.activities[0].description.length === 0
```

Comment:
Looks like this would work if the first activity is "Other," but what about if there are two activities and the second one is "Other?"

Reply:
I changed this PR, please take a look when you have time (it's yesterday's code).